# Used to publish a double-hand gesture pair and match status
std_msgs/Header header
string left_gesture
string right_gesture
bool match